R/PlatinumGenomes-QC/comparison/sexCheck/plinkCombined/README.md

```diff
@@ -0,0 +1,57 @@
+This directory contains the inputs and outputs of PLINK check-sex analysis.
+
+#### Inputs:
+  * [chrX-all-samples-merged.bim](chrX-all-samples-merged.bim) and [chrX-all-samples-merged.bed](chrX-all-samples-merged.bed):
+    * These files were transformed from a VCF file that was created using
+      the query included below. The primary reason for using a customized input
+      file was to ensure PLINK used the same input as BigQuery. For instance,
+      the 'expanded table' used in the query only contains non-variant segments
+      for SNPs (i.e. indels matching the reference are not included). Thus,
+      indels needed to be excluded in the provided inputs.
+      Also, to obtain accurate results in PLINK, the input files must
```

Comment:
We're using updated language now `expanded table` -> `multi-sample variants table` per http://biorxiv.org/content/early/2015/12/24/035295

Also, the multi-sample variants table does not include any non-variant segments.  What it does include are reference calls for a SNP for any samples with a non-variant segment that overlapped that SNP.

Suggested edit:
*For instance, the multi-sample variants table used in the query only contains reference-matches for SNPs (i.e. indels matching the reference are not included).*